commodore/helpers.py

```diff
@@ -109,7 +109,7 @@ def clean_working_tree(config: Config):
         rmtree = shutil.rmtree
     click.secho("Cleaning working tree", bold=True)
     rmtree(config.inventory.inventory_dir, ignore_errors=True)
-    rmtree(config.inventory.dependencies_dir, ignore_errors=True)
```

Comment:
This has the side-effect that Kapitan won't download HTTP(S) dependencies again if they already exist on disk. Which is ok-ish, I guess.

Should we maybe add a flag to the `cleanup` command to still be able to remove everything?

Reply:
No, in this case I just would also drop the `libs` dir. Downloading libs is not what takes time.

Reply:
That's not what I meant. Kapitan dependencies are usually downloaded to `dependencies/component-name/manifests/name.yaml` or similar.